packages/flutter_tools/lib/src/build_runner/resident_web_runner.dart

```diff
@@ -404,6 +453,130 @@ class ResidentWebRunner extends ResidentRunner {
     return OperationResult.ok;
   }
 
+  Future<int> _runExperimental({
```

Comment:
Consider having experimental and non-experimental implementations inheriting shared code from a common base class. The base class has a factory constructor that reads the feature flag and gives an instance of the correct derived class.

Reply:
Done